source/mainnet/net/guides/wallet-sdk/wallet-sdk-identity-creation.rst

```diff
@@ -0,0 +1,249 @@
+.. _wallet-sdk-identity-creation:
+
+====================
+Creating an identity
+====================
+
+To create an account on the Concordium blockchain, one must first acquire an identity. Therefore, as an initial step, a user of a wallet will always have to create an identity before being able to send account transactions.
+
+An identity is acquired by generating an identity request and sending it to an identity provider. The user will then be taken through the identity verification
+process that is specific for that chosen identity provider. This happens outside of the wallet application.
+
+* `Creating an identity request`_
+* `Sending an identity request`_
+* `Retrieving the identity after creation`_
+
+++++++++++++++++++++++++++++
+Creating an identity request
+++++++++++++++++++++++++++++
+
+The first step is to create the actual identity request. To do this, you need the list of identity providers. Refer to :ref:`Identity Provider List` to understand how to retrieve it.
+
+.. tabs::
+
+    .. tab::
+
+        TypeScript (Web)
+
+        .. code-block:: javascript
+
+              import {
+                ConcordiumGRPCWebClient,
+                ConcordiumHdWallet,
+                createIdentityRequestWithKeys,
+                IdentityRequestWithKeysInput,
+                IdObjectRequestV1,
+                Versioned,
+            } from '@concordium/web-sdk';
+
+            // Select the identity provider that is to be used for creating the identity. Here we
+            // choose the first one in the list available from the Concordium wallet-proxy.
```

Comment:
The font on the comments are very annoying(?) to read. :thinking:

Reply:
I don't know if that can be changed, but I also don't think it is that bad.